plugin-api/src/main/java/org/sonar/api/utils/log/Loggers.java

```diff
@@ -21,22 +21,14 @@
 
 public abstract class Loggers {
```

Comment:
Since all the logging API in sonar-plugin-api will just wrap slf4j without providing any log interception for tests, couldn't we deprecate it and recommend everyone to move to using slf4j directly?

Reply:
That's my next plan :) but I wanted to keep the door open since we have the `setLevel` use case to sort out. So for now, no deprecation.

Reply:
Ok. The javadoc mentions :The standard use-case is to customize logging of embedded 3rd-party libraries.". I guess that's a valid use case, and the plugin would have to get the logger for the 3rd party lib class to change its level.

But I wonder how much it's actually used. I couldn't find any uses of it in our internal plugins. It's used by SonarQube, but SonarQube and other products can do that directly with logback.